flink-test-utils-parent/flink-connector-test-utils/src/main/java/org/apache/flink/connector/testframe/testsuites/SourceTestSuiteBase.java

```diff
@@ -86,6 +103,8 @@
 public abstract class SourceTestSuiteBase<T> {
 
     private static final Logger LOG = LoggerFactory.getLogger(SourceTestSuiteBase.class);
+    static ExecutorService executorService =
+            Executors.newFixedThreadPool(Runtime.getRuntime().availableProcessors() * 2);
```

Comment:
`Runtime.getRuntime().availableProcessors() * 2` is too expensive from my side, and could you add a `private` decoration too.

Reply:
I will remove this field in this class.